holidays/locale/th/LC_MESSAGES/MY.po

```diff
@@ -248,3 +248,7 @@ msgstr "วันอีฎิ้ลฟิตริ (เพิ่มเติม)
 #. Additional holiday in commemoration of the 2017 SEA Games.
 msgid "Cuti tambahan sempena memperingati SAT 2017"
 msgstr "วันหยุดเพิ่มเติมเนื่องในโอกาสการแข่งขันกีฬาซีเกมส์ปี 2017"
+
+#. Eid al-Fitr (Third Day).
+msgid "Hari Raya Puasa (Hari Ketiga)"
+msgstr "วันอีฎิ้ลฟิตริวันที่สาม"
```

Comment:
@PPsyrius, I hope this is a correct translation, but please take a look.

Reply:
Yes, this is the correct translation 👍